bin/jade.js

```diff
@@ -129,7 +129,9 @@ if (files.length) {
       process.exit(1);
     });
   } else {
-    files.forEach(renderFile);
+    files.forEach(function (file) {
+      renderFile(file);
+    });
```

Comment:
Because forEach passes index as second argument, which is unwanted in this case.